src/config_map.c

```diff
@@ -107,8 +107,9 @@ const CONFIG_OPTION g_ConfigOptionMap[] = {
     { .name = "walk_to_items",                  .type = COT_BOOL,   .target = &g_Config.walk_to_items,                       .default_value = &(bool){false},                      0},
     { .name = "disable_trex_collision",         .type = COT_BOOL,   .target = &g_Config.disable_trex_collision,              .default_value = &(bool){false},                      0},
     { .name = "start_lara_hitpoints",           .type = COT_INT32,  .target = &g_Config.start_lara_hitpoints,                .default_value = &(int32_t){LARA_HITPOINTS},          0},
-    { .name = "healthbar_showing_mode",         .type = COT_ENUM,   .target = &g_Config.healthbar_showing_mode,              .default_value = &(int32_t){BSM_FLASHING_OR_DEFAULT}, .param = m_BarShowingModes},
-    { .name = "airbar_showing_mode",            .type = COT_ENUM,   .target = &g_Config.airbar_showing_mode,                 .default_value = &(int32_t){BSM_DEFAULT},             .param = m_BarShowingModes},
+    { .name = "healthbar_showing_mode",         .type = COT_ENUM,   .target = &g_Config.healthbar_show_mode,                 .default_value = &(int32_t){BSM_FLASHING_OR_DEFAULT}, .param = m_BarShowModes},
+    { .name = "airbar_showing_mode",            .type = COT_ENUM,   .target = &g_Config.airbar_show_mode,                    .default_value = &(int32_t){BSM_DEFAULT},             .param = m_BarShowModes},
+    { .name = "enemy_healthbar_showing_mode",   .type = COT_ENUM,   .target = &g_Config.enemy_healthbar_show_mode,           .default_value = &(int32_t){BSM_ALWAYS},              .param = m_BarShowModes},
```

Comment:
I think the `name` and `target` are not matching?

Reply:
https://github.com/LostArtefacts/TR1X/pull/1331#discussion_r1573911299 has the fix, the lang keys aren't matching up.

Reply:
The json key has to stay for compatibility reasons, the rest should use the better naming.